tests/mock_crust/churn.rs

```diff
@@ -90,6 +91,40 @@ fn did_receive_get_success(node: &TestNode,
     }
 }
 
+fn quorum_calculate(nodes: &[TestNode],
+                    request_info: (Authority, Authority, DataIdentifier, MessageId),
+                    target: Authority,
+                    send_get: bool,
+                    check_received: bool)
+                    -> (usize, usize) {
+    let group = nodes.iter()
+        .filter(|n| n.routing_table().is_recipient(&target.to_destination()))
+        .collect_vec();
+    let quorum = (group.len() * QUORUM - 1) / 100 + 1;
+
+    if send_get {
+        for node in &group {
+            let _ = node.inner.send_get_request(request_info.0,
+                                                request_info.1,
+                                                request_info.2,
+                                                request_info.3);
+        }
+    }
+
+    let num_received = if check_received { 
+        group.iter()
+             .filter(|node| did_receive_get_request(node,
+                                                    request_info.0,
+                                                    request_info.1,
+                                                    request_info.2,
+                                                    request_info.3))
+             .count()
+    } else {
+        0
+    };
+    (quorum, num_received)
+}
+
```

Comment:
We never actually set both `send_get` and `check_received` to true - looks like those parts would make more sense as separate functions.

I'd see the first two `let`s in a function like `target_group_quorum`, and then the sending and checking functions could call it.